pokemongo_bot/__init__.py

```diff
@@ -1059,14 +1060,14 @@ def get_pos_by_name(self, location_name):
             possible_coordinates = re.findall(
                 "[-]?\d{1,3}[.]\d{3,7}", location_name
             )
-            if len(possible_coordinates) >= 2:
+            if len(possible_coordinates) == 2:
                 # 2 matches, this must be a coordinate. We'll bypass the Google
                 # geocode so we keep the exact location.
                 self.logger.info(
                     '[x] Coordinates found in passed in location, '
                     'not geocoding.'
                 )
-                return float(possible_coordinates[0]), float(possible_coordinates[1]), (float(possible_coordinates[2]) if len(possible_coordinates) == 3 else self.alt)
+                return float(possible_coordinates[0]), float(possible_coordinates[1]), self.alt
```

Comment:
@BreezeRo Could you change line 1063 and 1070 back? This was merged in PR #4581.